str/trtools/merge_str_runner.py

```diff
@@ -95,8 +95,12 @@ def main(
         if len(cpg_ids) != len(set(cpg_ids)):
             raise ValueError('Duplicate CPG IDs detected in sample list')
 
+        # make batch_vcfs relative to BATCH_TMP/inputs
+        batch_vcfs = [vcf.split('inputs/')[1] for vcf in batch_vcfs]
```

Comment:
I'm not sure if you can run a `.split()` on a ResourceFile directly, or whether you'll need to do that on the String representation of the same... It may work?
```suggestion
        batch_vcfs = [str(vcf).split('inputs/')[1] for vcf in batch_vcfs]
```